### DISCUSSION TITLE
How does search for covers work?

### DISCUSSION THREAD

**Discussion by mateusz-bak** [PaulWoitaschek/Voice#2353] [Q&A]
Hi @PaulWoitaschek could you share some details on how you implemented search for covers online?

I see that you use duckduckgo for that but I'm curious how it is actually implemented as they don't have any api for images search.

Could you share how your app get's the results please?

I'm the maintainer of https://github.com/mateusz-bak/openreads and would really appreciate your help on this as my app would benefit a lot from this feature.

**Comment by ghost** [comment]
https://github.com/PaulWoitaschek/Voice/tree/main/cover
